lib/packages.cf

```diff
@@ -32,13 +32,21 @@ bundle common package_module_knowledge
     windows::
       "platform_default" string => "msiexec";
 @endif
+@if minimum_version(3.16)
+    termux::
```

Comment:
I don't think it is necessary to guard this based on minimum version.

termux class may not be defined until 3.16.1, but that just means it will be out of context, so won't apply.

Reply:
Actually, i do think you need this, but not for 3.16, more for 3.12.2? whenever interpreter was introduced.